rts/Lua/LuaHandle.cpp

```diff
@@ -769,6 +769,28 @@ void CLuaHandle::GameFrame(int frameNum)
 	RunCallInTraceback(L, cmdStr, 1, 0, traceBack.GetErrFuncIdx(), false);
 }
 
+/*** Called at the end of every game simulation frame (30 per second).
```

Comment:
I wouldn't write the 30 down here because hopefully sim framerate would become configurable at some point in the future and then there would be these 30's spread all over the place being incorrect.